src/teleop/gui/src/components/ISHTask.vue

```diff
@@ -1,20 +1,53 @@
 <template>
 <div>
     <p>ISH</p>
+    <div class="box camera">
+      <Cameras v-bind:numCams="4" v-bind:primary="true" v-bind:mission="'Science'" v-bind:channel="'/cameras_control_ish'"/>
+    </div>
 </div>
 </template>
 
 <script>
 
+import Cameras from '../components/Cameras.vue';
+
 export default {
   data() {
     return {
 
     }
   },
 
+  methods: {
+    publish : function(){
+
+    }
+  },
+
   components:{
+    Cameras
 }
 }
 </script>
 
+<style>
+
+  :root {
```

Comment:
While I of course appreciate some good CSS this seems unfit for this PR being that this is CSS for the ISH GUI. Best course of action might be to create a wiki page that documents the styling that you've been doing with your components so other people can try to match it and the GUI can be more cohesively styled